canard.h

```diff
@@ -351,8 +365,8 @@ void canardSetLocalNodeID(CanardInstance* ins,
                           uint8_t self_node_id);
 
 /**
- * Returns node ID of the local node.
- * Returns zero (broadcast) if the node ID is not set, i.e. if the local node is anonymous.
+ * enqueueTxFrames
+ * enqueueTxFramesD is not set, i.e. if the local node is anonymous.
```

Comment:
![image](https://user-images.githubusercontent.com/3298404/55099572-99787380-50c8-11e9-82e5-194e69871c77.png)

Reply:
Am I the only one that gets weird things happening to random places in my code all the time when using vscode?

Reply:
I have never experienced any issues with vscode.